examples/generic_connector_configs/elasticsearch_secretless.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+services:
+  token-service:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:9030
+    credentials:
+      token:
+        from: keychain
+        get: summon#elasticsearch/access_token
```

Comment:
I wonder if something like` path/to/elasticsearch/access/token` would be more informative for the example.

Reply:
Agreed - this may be a change we should make in all the examples. @izgeri , what do you think?

Reply:
doesn't much matter to me, but this is the syntax for the keychain provider so we may consider switching the examples to a different provider in that case